### ISSUE TITLE
[Request] Game Boy Pocket device type

### ISSUE THREAD

**Issue by Sanaki** [LIJI32/SameBoy#410]
Would it be trivial to add Game Boy Pocket to SameBoy as a device type? Someone just cracked out homebrew that depends on that being available. I honestly didn't even realize it identified itself as different from the DMG previously.
https://drludos.itch.io/gb-corp

We have an achievement developer wanting to make a set for the game, but thus far it seems no GB libretro core supports the GBP as an emulated device. I figure this seems right up your alley.

**Comment by LIJI32** [comment]
Yeah, although there are a few other differences I documented over the years.
